test/basic-querying.test.js

```diff
@@ -172,7 +177,7 @@ describe('basic-querying', function() {
       User.find({skip: 1, limit: 4, order: 'seq'}, function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users[0].seq.should.be.eql(1);
+        if (!isCassandraConnector) users[0].seq.should.be.eql(1);
```

Comment:
To support pagination with Cassandra, it may require changes in the response of `find`, which is an array of matching records today. Cassandra probably need to return `pagingState` as well and it turns the response into an object such as `{pagingState: ..., data: [...]}`. Maybe the `options.paging` arg can be used to indicate such requirement.